### DISCUSSION TITLE
Bird API Rate Limit

### DISCUSSION THREAD

**Discussion by jacobwhite** [ubahnverleih/WoBike#241] [General]
Do you know if there's a rate limit on the Bird API? I have a private bird that I'd like to track statistics on. I have it updating every 5 minutes, but that's not super useful for logging trips with the GPS data.

**Comment by BastelPichi** [comment]
The bird app calls about every 5 seconds, requesting every second should be fine. Im unsure about private birds in the API, can you tell your experiences?
